backend/hypothetical_books_backend/urls.py

```diff
@@ -14,16 +14,16 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.urls import include, path
-
-api_version = 1
-api_prefix = f'api/v{api_version}/'
+from django.conf import settings
+from django.conf.urls.static import static
+from .version import API_PREFIX
 
 urlpatterns = [
-    path(f'{api_prefix}books', include('books.urls')),
-    path(f'{api_prefix}genres', include('genres.urls')),
-    path(f'{api_prefix}vendors', include('vendors.urls')),
-    path(f'{api_prefix}sales', include('sales.urls')),
-    path(f'{api_prefix}purchase_orders', include('purchase_orders.urls')),
-    path(f'{api_prefix}auth', include('authapp.urls', namespace='authentication')),
-    path(f'{api_prefix}buybacks', include('buybacks.urls')),
-]
+    path(f'{API_PREFIX}books', include('books.urls')),
+    path(f'{API_PREFIX}genres', include('genres.urls')),
+    path(f'{API_PREFIX}vendors', include('vendors.urls')),
+    path(f'{API_PREFIX}sales', include('sales.urls')),
+    path(f'{API_PREFIX}purchase_orders', include('purchase_orders.urls')),
+    path(f'{API_PREFIX}auth', include('authapp.urls', namespace='authentication')),
+    path(f'{API_PREFIX}buybacks', include('buybacks.urls')),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
What is being added to the `urlpatterns` list here and why?

Reply:
https://docs.djangoproject.com/en/4.1/howto/static-files/

Reply:
static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) is `'^api/v1/static/(?P<path>.*)$'`

Reply:
It seems to say that this only works in debug mode. On prod we aren't using debug mode. Will this be an issue?

Reply:
`--insecure` flag will be used with runserver so it is actually serving static files. In production we would serve these static files using Nginx but this is an easier path

Reply:
Ok. As long as this isn't an issue within the code, then I'll approve